rotkehlchen/tests/utils/history.py

```diff
@@ -1159,15 +1164,26 @@ def maybe_mock_historical_price_queries(
         should_mock_price_queries: bool,
         mocked_price_queries,
         default_mock_value: Optional[FVal] = None,
+        dont_mock_price_for: Optional[List['Asset']] = None,
 ) -> None:
     """If needed will make sure the historian's price queries are mocked"""
     if not should_mock_price_queries:
         return
 
+    if dont_mock_price_for is None:
+        dont_mock_price_for = []
+
+    # save the mocked behaviour in this variable to be used when
+    # the list of assets to not mock is non empty.
+    old_function = historian.query_historical_price
+
     def mock_historical_price_query(from_asset, to_asset, timestamp):
         if from_asset == to_asset:
             return FVal(1)
 
+        if from_asset in dont_mock_price_for:
+            return old_function(from_asset, to_asset, timestamp)
```

Comment:
nice. This is exactly what I would have suggested in discord when you asked me. Wasn't sure it would work though :rofl: